source/write-scripts/require-external-modules.txt

```diff
@@ -108,6 +114,19 @@ install the modules either:
 - Globally
 - In the ``node_modules`` directory in your current working directory.
 
+.. tip::
+
+   If you package your script as a :ref:`snippet <snip-overview>`, you
```

Comment:
This is not just the case for snippets, but for all mongosh scripts.

Reply:
Done